django/db/migrations/questioner.py

```diff
@@ -188,6 +192,24 @@ def ask_rename(self, model_name, old_name, new_name, field_instance):
         return self._boolean_input(msg % (model_name, old_name, model_name, new_name,
                                           field_instance.__class__.__name__), False)
 
+    def ask_small_to_big_autofield(self, field_name, model_name):
+        """Upgrading an AutoField to a BigAutoField."""
+        if not self.dry_run:
+            choice = self._choice_input(
+                "You are changing the field {}.{} from a SmallAutoField to a BigAutofield "
+                "(perhaps as a result of upgrading Django). The database needs to upgrade from int to bigint, "
+                "which may take a while, if you have a lot of rows ".format(model_name, field_name),
+                [
+                    "Create upgrade migration (may take a while to apply)",
+                    "Quit, and let me specify SmallAutoField in models.py",
```

Comment:
Saying `models.py` isn't always accurate, some apps have their models module split up. Perhaps `"...let me specify SmallAutoField in the {} definition".format(model_name)` is better.

Reply:
@adamchainz can i add you to my fork and you can help me clean this up?